fred/fred_commands/__init__.py

```diff
@@ -45,11 +45,12 @@ async def on_command_error(self, ctx: commands.Context, error):
             return
         elif isinstance(error, commands.BadArgument):
             self.logger.info("Successfully deferred error of bad argument")
-            _, target_type, _, missing_argument_name, *_ = str(error).split('"')
-            output = f"At least one parameter for this command was entered incorrectly: '{missing_argument_name}'"
-            if target_type:
-                output += f" of type '{target_type}'"
-            await ctx.reply(output)
+            # self.logger.debug(error)
+            # _, target_type, _, missing_argument_name, *_ = str(error).split('"')
+            # output = f"At least one parameter for this command was entered incorrectly: '{missing_argument_name}'"
+            # if target_type:
+            #     output += f" of type '{target_type}'"
```

Comment:
Did you mean to push this comment ?

Reply:
This was commented out because it was broken, but I'm not sure if I should get rid of it just yet.

Reply:
Let's keep it, it should get fixed